packages/jest-snapshot/src/InlineSnapshots.ts

```diff
@@ -49,10 +49,15 @@ export function saveInlineSnapshots(
   snapshots: Array<InlineSnapshot>,
   prettierPath: Config.Path,
 ): void {
-  const prettier = prettierPath
-    ? // @ts-expect-error requireOutside Babel transform
-      (requireOutside(prettierPath) as Prettier)
-    : undefined;
+  let prettier: Prettier | null = null;
+  if (prettierPath) {
+    try {
+      // @ts-expect-error requireOutside Babel transform
+      prettier = requireOutside(prettierPath) as Prettier;
+    } catch {
```

Comment:
thinking about it, we should probably just swallow `MODULE_NOT_FOUND` and rethrow other errors. Meh 🤷